lib/sles4sap/sap_deployment_automation_framework/deployment_connector.pm

```diff
@@ -387,3 +389,17 @@ sub destroy_orphaned_resources {
     record_info('az destroy', "Following orphaned resources will be destroyed:\n" . join("\n", @orphaned_resources));
     destroy_resources(timeout => $args{timeout}, resource_cleanup_list => \@orphaned_resources);
 }
+
+=head2 no_cleanup_tag
+
+    no_cleanup_tag();
+
+Returns tag name that marks resource to be omitted during cleanup routine. Tag name can be defined by OpenQA setting
+`SDAF_NO_CLEANUP_TAG` with default value being 'sdaf_cleanup_ignore'.
+This function ensures default value naming consistency across all modules, instead defining it with each 'get_var' call.
+
+=cut
+
+sub no_cleanup_tag {
+    return get_var('SDAF_NO_CLEANUP_TAG', 'sdaf_cleanup_ignore');
```

Comment:
I am using generic name for default value `sdaf_cleanup_ignore` , but in the job group we will set 'pcw_ignore' so it matches with what PCW expects.